python-scripts/src/main/python/dataproducts/util/utils.py

```diff
@@ -547,7 +568,16 @@ def get_content_plays(result_loc_, start_date_, end_date_, druid_, config_):
     query = query.replace('$end_date', end_date_.strftime('%Y-%m-%dT00:00:00+00:00'))
     query = query.replace('$app', config_['context']['pdata']['id']['app'])
     query = query.replace('$portal', config_['context']['pdata']['id']['portal'])
-    response = requests.post(url, data=query, headers=headers)
+    query = json.loads(query)
+    if start_date_ > datetime(2020, 7, 2):
+        query['filter']['fields'].append(
+            {
+                "type": "in",
+                "dimension": "content_mimetype",
+                "values": config_['content']['mimeType']
+            },
+        )
+    response = requests.post(url, data=json.dumps(query), headers=headers)
```

Comment:
this logic is not required. Let the job take in a parameter for the query version to run and the invoker of the job should know the version to use